CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.14.7)
 
 project(remotechargeport
-    VERSION 2024.02.0
+    VERSION 2024.05.0
```

Comment:
EVerest has only one digit for month (2024.5.0). Should we adhere to that?

Reply:
No, I don't like the EVerest scheme with a single digit - it doesn't sort well...

Reply:
Good point